rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -8,6 +8,34 @@
  */
 package org.eclipse.sw360.rest.resourceserver.restdocs;
 
+import static org.eclipse.sw360.datahandler.thrift.MainlineState.MAINLINE;
```

Comment:
Resorting the imports looks like an unnecessary refactoring for me. And what IDE do you use, I thin we usually sort the static imports to the bottom of the list and AFAIK this is also the way IntelliJ handles the imports?

Reply:
For this case I have reverted the import reordering.